index.js

```diff
@@ -150,4 +173,4 @@ function expand(input) {
   var file = path.basename(input);
 
   return dirname + '/{' + file + '}';
-}
+}
```

Comment:
We should preserve newline at end of file

Reply:
Fixed